pkg/ui/src/views/statements/barCharts.module.styl

```diff
@@ -0,0 +1,69 @@
+
+
+@require '~styl/base/palette.styl'
+@require '~src/components/core/index'
+
+.bar-chart
```

Comment:
I dig this whole approach. And I'm excited to see us breaking components up this way since it will make it easier to reuse in the app. And if / when we want to use a component in another app, it will make it easier to move and publish.

One thing I look forward to with CSS Modules is losing the pesky BEM class naming since it no longer provides a function. I understand why you wouldn't do it here in the WIP, but I just wanted to call out that I think it's redundant to have the nesting under `.bar-chart`. So styles could become something like `${styles["bar"]}` instead of `${styles["bar-chart__bar"]}`

Reply:
@nathanstilwell , I totally agree about BEM formatting, I preserved current names to:
- minimize code changes (easy to review and track lots of changes)
- to be able find code duplications which still remain in original stylings